libbeat/outputs/elasticsearch/client.go

```diff
@@ -674,6 +681,20 @@ func (client *Client) String() string {
 	return "elasticsearch(" + client.Connection.URL + ")"
 }
 
+func (client *Client) Connect() error {
+	err := client.Connection.Connect()
+	if err != nil {
+		return err
+	}
+
+	if strings.HasPrefix(client.GetVersion(), "7.") {
```

Comment:
I wonder if we should go a bit more sophisticated here and actually check that the major version is > 6 by using our internal version package. Like this we don't have it breaking by accident as soon as we have master based on 8.

Reply:
Did change version handling in general, such that we always have a 'parsed' version available. => Changed logic to use `>= 7`.